README.md

```diff
@@ -13,6 +13,8 @@ and generating an archive of DDL metadata or logs for consumption by Assessment
 or the Translation Service. For more information, [read the tool
 documentation](https://cloud.google.com/bigquery/docs/generate-metadata).
 
+    The Dumper requires Java 8 or higher. You can check whether you have a recent enough version of Java installed by running the command `java -version`.
```

Comment:
The indentation seems off.

Reply:
This is intended to align with the bullet-point above. You can click the "View file" in the actions menu (three dots "...") on the right to see the preview of the rendered page.